docs/management/cases/cases.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[cases]]
+== Cases
+
+preview::[]
+
+Cases are used to open and track issues directly in the {kib}. All cases list
+the original reporter and all the users who contribute to a case (_participants_).
+You can also send cases to third party systems by configuring external connectors.
+
+[role="screenshot"]
+image::images/cases.png[Cases page]
+
+NOTE: If you create cases in the {observability} or {security-app}, they are not
```

Comment:
Maybe we can add also that you cannot attach alerts from Observability or Security to a case.